Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM ubuntu:14.04
+FROM ubuntu:17.10
 MAINTAINER dev@dwolla.com
 USER root
 
-ENV SECURITY_MONKEY_VERSION=v0.8.0
+ENV SECURITY_MONKEY_VERSION=v0.9.1
 
 RUN apt-get update &&\
-    apt-get -y install python-pip python-dev python-psycopg2 libpq-dev git libffi-dev &&\
+    apt-get -y install python-pip python-dev python-psycopg2 libpq-dev git libffi-dev python-virtualenv libssl-dev &&\
     cd /usr/local/src &&\
     git clone --depth 1 --branch $SECURITY_MONKEY_VERSION https://github.com/Netflix/security_monkey.git &&\
     cd security_monkey &&\
+    virtualenv venv &&\
```

Comment:
What impact does putting python in a virtualenv have on the files that are installed in `/usr/local/src/security_monkey`?